src/components/CategoriesBrowser/index.css

```diff
@@ -0,0 +1,48 @@
+:root {
+  --white: #ffffff;
+  --black: #000000;
+  --navy: #1b1f24;
+  --light-navy: #292f36;
+  --dark-gray: #434b56;
+  --gray: #646e7b;
+  --light-gray: #d1d2d3;
+  --tag-gray: #f6f6f6;
+  --yellow: #ffce51;
+  --orange: #f59e33;
+  --dark-orange: #df6941;
+  --tag-orange: #fef3e7;
+  --dark-blue: #0064af;
+  --blue: #009dcf;
+  --tag-blue: #edf0ff;
+  --dark-green: #3ab66e;
+  --green: #add960;
+  --turquoise: #42c2b3;
+  --tag-turquoise: #d9f3f0;
+  --dark-red: #8f1233;
+  --red: #e13f58;
+}
```

Comment:
They already there. If I understand correct it is specific of PostCSS (tachyons uses it in the same way). Probably there is some postcss plugin which allows to share variables across the files. Will investigate it tomorrow. But it will not appear in result CSS.
for example: https://github.com/tachyons-css/tachyons/blob/master/src/_border-colors.css